### PR TITLE
Upgrade the driver to version 2.0.0-rc.1

### PR DESCRIPTION
**Pull request by othonrsanchez**
Upgrade the driver to version 2.0.0-rc.1

With this commit we introduce multiple changes to the driver to make
it easier to use:
* Added the `getTableNames` method to the `QldbDriver` class.
* Session pooling functionality moved to `QldbDriver`.
* Upgraded dependency from the AWS SDK for Java v1 to the AWS SDK for Java v2.
* Fixed the issue where two different Ion Java packages were present in the classpath.
* `PooledQldbDriver` has been removed. Please use `QldbDriver` instead.
* `QldbSession` and `Transaction` classes have been removed.  Please use
`QldbDriver` instead.
* `QldbDriver.execute(String)` method has been removed as we found it made the
driver confusing about when to use one execute method over the other. Please use `QldbDriver
.execute(ExecutorNoReturn executor)` method  or `QldbDriver.execute(Executor executor)` instead.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: buildspec.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+version: 0.2
```
Comment (yohanmartin) [inline buildspec.yml:1]: Do we need this file?

Comment (othonrsanchez) [reply buildspec.yml:1]: no, we don't. removing it.

File: .gitignore [hunk 2]
```diff
@@ -1,28 +1,9 @@
...
-*.war
-*.nar
-*.ear
-*.zip
-*.tar.gz
-*.rar
-
-# IDE configurations
-*.iml
-.vscode/
```
[the remark this answers is not in the document]
Comment (othonrsanchez) [reply .gitignore:17]: will revert

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,23 +1,75 @@
-# Changelog
-All notable changes to this project will be documented in this file.
+# Changelog All notable changes to this project will be documented in this file.
```
[the remark this answers is not in the document]
Comment (othonrsanchez) [reply CHANGELOG.md:1]: fair. will remove

**Pull request merged**